internal/cmd/io.go

```diff
@@ -57,13 +63,40 @@ func NewUpstreamIO(ctx context.Context, id, stdout, stderr, stdin string, termin
 
 	// Create IO for binary logging driver.
 	if u.Scheme != "binary" {
-		return nil, errors.Errorf("scheme must be 'binary', got: '%s'", u.Scheme)
+		return nil, fmt.Errorf("scheme must be 'binary', got: '%s'", u.Scheme)
 	}
 
 	return NewBinaryIO(ctx, id, u)
 }
 
+// isClosedIOErr checks if the error is from the underlying file or pipe already being closed.
+func isClosedIOErr(err error) bool {
+	for _, e := range []error{
+		os.ErrClosed,
+		net.ErrClosed,
+		io.ErrClosedPipe,
+		winio.ErrFileClosed,
+		hcs.ErrAlreadyClosed,
```

Comment:
Should `io.EOF` be included in this set?

Reply:
only for completeness, since `io.Copy` will suppress `io.EOF` errors from the source